src/pspm_pupil_pp.m

```diff
@@ -169,7 +144,7 @@
 end
 %% 4 Load
 action_combine = ~strcmp(options.channel_combine, 'none');
-[lsts, data] = pspm_load_channel(fn, options.channel, 'pupil');
+[lsts, data,] = pspm_load_channel(fn, options.channel, 'pupil');
```

Comment:
I checked load_channel and I can understand the output variables are flexible (can be two or more). I tested `[lsts, data] =` and `[lsts, data,] =` and found both of them working. However I find this comma seems to be added intentionally. I dont quite know why. I checked other functions and saw another [a,b,] appear at line 117 in `scr_pp`. Maybe it is a minor stuff, just to explain my question here.

Reply:
This was a typo and is removed now.